Core/HLE/sceKernelMemory.cpp

```diff
@@ -2301,3 +2301,96 @@ void Register_SysMemUserForUser()
 {
 	RegisterModule("SysMemUserForUser", ARRAY_SIZE(SysMemUserForUser), SysMemUserForUser);
 }
+
+
+struct HeapInformation : public KernelObject {
+	HeapInformation() {}
+	int uid;
+	int partitionId;
+	u32 size;
+	int flags;
+	u32 address;
+	std::string name;
+	BlockAllocator alloc;
+	static int GetStaticIDType() { return SCE_KERNEL_TMID_Fpl; }//  wrong
+	int GetIDType() const override { return SCE_KERNEL_TMID_Fpl; }//  wrong
```

Comment:
See also `PPSSPP_KERNEL_TMID_Module` and similar for using fake TMID values.  For savestates to work correctly, we need unique values right now.

-[Unknown]